### PR TITLE
procs: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

# ToDo

- [x] Standardize descriptions
- [x] Fix description errors such as mentioning unused things in code
- [x] Remove wrong mnemonics

### CODE REVIEW

File: pages/linux/procs.md [hunk 1]
```diff
@@ -7,18 +7,18 @@
...
 
 `procs {{zsh}}`
 
-- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
+- List information about all processes sorted by CPU time in ascending or descending order:
 
 `procs {{--sortd|--sorta}} cpu`
 
-- Show information about processes with either a PID, command, or user containing (`zsh` or `firefox`):
+- List information about processes with either a PID, command, or user containing `firefox`:
```
Comment (pixelcmtd) [inline pages/linux/procs.md:18]: I'm not sure if I fully understand the semantics of this, but isn't it more like this

```suggestion
- List information about processes with either a PID, command, or user containing `41` or `firefox`:
```

File: pages/linux/procs.md [hunk 2]
```diff
@@ -7,18 +7,18 @@
...
 
-- Show information about processes, if the commands which started them contain `zsh`:
+- List information about processes, if the commands which started them contain `zsh`:
 
 `procs {{zsh}}`
 
-- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
+- List information about all processes sorted by CPU time in ascending or descending order:
 
 `procs {{--sortd|--sorta}} cpu`
```
Comment (kbdharun) [inline pages/linux/procs.md:16]:
```suggestion
`procs {{--sorta|--sortd}} cpu`
```
Shouldn't it be this? I am currently busy I will look into the review requests later today.

Comment (EmilySeville7cfg) [reply pages/linux/procs.md]: Nice catch, yes, it should. 😄

File: pages/linux/procs.md [hunk 3]
```diff
@@ -7,18 +7,18 @@
 
 `procs`
 
-- Show information about processes, if the commands which started them contain `zsh`:
+- List information about processes, if the commands which started them contain `zsh`:
 
 `procs {{zsh}}`
 
-- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
+- List information about all processes sorted by CPU time in ascending or descending order:
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply pages/linux/procs.md:14]: Then let's decline mnemonics removal for long options. I mean allow to create mnemonics for everything.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**